src/components/Common/UI/Alert/AlertTypes.ts

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode, ComponentType } from 'react'
+import type { SVGProps } from 'react'
+
+export interface AlertProps {
+  /** The variant of the alert */
+  variant?: 'info' | 'success' | 'warning' | 'error'
+  /** The label text for the alert */
+  label: string
```

Comment:
Nit:  wondering if label has too many associations for form elements? that might be overthinking it tho. alternative i could think of would be title or heading but those are also pretty loaded

Reply:
I think i'm ok with label because we could use `aria-label` on the alert role so it does align with ARIA.